python/test/openmldb_client_test.py

```diff
@@ -166,6 +168,18 @@ def execute_insert_sqls(self, connection, sqls):
         connection.execute(sql);
       except Exception as e:
         self.assertTrue(False)
+  
+  def check_sqlalchemy_table_api(self, engine):
```

Comment:
We may move these in seperated file later.

Reply:
updated.